Jenkinsfile

```diff
@@ -41,10 +30,26 @@ node('vagrant') {
 
         EcoSystem ecoSystem = new EcoSystem(this, "gcloud-ces-operations-internal-packer", "jenkins-gcloud-ces-operations-internal")
 
-        try {
+        stage('Checkout') {
+            checkout scm
+        }
+
+        stage('Lint') {
+            lintDockerfile()
+            shellCheck("resources/startup.sh resources/upgrade-notification.sh resources/pre-upgrade.sh")
 
+            if (env.CHANGE_TARGET) {
+                echo 'This is a pull request; checking changelog...'
+                String newChanges = changelog.changesForVersion('Unreleased')
+                if (!newChanges || newChanges.allWhitespace) {
+                    unstable('There is no information about unreleased changes in the CHANGELOG.md')
```

Comment:
Because of the double negation (no information, unreleased changes) I'd like to discuss a slightly different text in order to minimize misunderstanding:

My suggestion:

> The CHANGELOG.md should contain new change entries but none were found.

Reply:
Yes, good idea. This way it's easier to understand.
I'm still thinking about letting the user know what is to do, which is filling the `Unreleased` section. How can this hint be included?

Reply:
> The CHANGELOG.md should contain new change entries in the `[Unreleased` section ]but none were found.